sdk/cosmos/azure-spring-data-cosmos/src/test/resources/application.properties

```diff
@@ -13,3 +13,13 @@ perf.acceptance.percentage=10
 
 # Populate query metrics
 cosmosdb.populateQueryMetrics=true
+
+cosmosdb.secondary.uri=${SECONDARY_ACCOUNT_HOST}
```

Comment:
To make it work with the current CI, you will also have to add these `SECONDARY_ACCOUNT_HOST` as output values in `test-resources.json` file
So as to output these values when we create cosmos db account.

Reply:
> @zhoufenqin - thank you for doing this change, I have added some comments, please take a look at them.
> 
> Also, the issue that you are trying to fix is different from the issue that you have targeted in this PR.
> 
> This change will allow user to support 2 different database accounts.
> 
> Whereas, this issue : [microsoft/spring-data-cosmosdb#520](https://github.com/microsoft/spring-data-cosmosdb/issues/520)
> is about supporting different databases in same cosmosdb account.
> 
> Single cosmosdb account can have multiple databases, and that's what the issue is.
> 
> Though this fix is also important to have - as it allows the end user to support two different cosmosdb accounts.
> 
> @saragluna @jialindai - this is the same conversation we had over email few weeks back where I explained the above problem.

hi @kushagraThapar, actually this PR can support "Single account multiple databases" and "multiple cosmos account", but now each cosmos template will create a cosmos client, so I'll research whether we have a better method to support "Single account multiple databases", reference [MongoTemplate](https://github.com/spring-projects/spring-data-mongodb/blob/master/spring-data-mongodb/src/main/java/org/springframework/data/mongodb/core/MongoTemplate.java#L204)

Reply:
I see, but the problem is that CosmosClient is supposed to be a singleton bean - because otherwise it may create multiple issues of performance and conflicting issues - since Cosmos Java SDK is a very thick client and it has business logic for cosmos db partitioning, routing caches, address caches and so on. 

Is there a way to achieve this without creating multiple instances of CosmosClient bean ?

Reply:
hi @kushagraThapar, We plan to finish this feature by two steps previously 
1: support multiple cosmos account  (this PR mainly for this)
2: support same account multiple database (we plan to optimize the CosmosClient)
So do you think we need to merge these steps in a single PR or we can separate these features in two PRs

Another problem: I think multiple cosmos account need multiple CosmosClient, their cosmos configuration are different, do you think so?

Reply:
@zhoufenqin Yes, you got it right, lets complete it in two steps: 
1. "Support multiple cosmos accounts" - which means we will certainly need multiple cosmos clients. Because one cosmos client can only support one cosmos account. So we are good on this PR for that. 
2. However, regarding "Support same account with multiple databases" - this is very common case and will be used by a lot of customers, so we will need to optimize it a bit, or find a way to do it with a single CosmosClient - because it should be able to support multiple database. This can be done in a later PR. 

However, before merging this PR, it will be better to merge the big PR - so that we don't have to resolve major conflicts - https://github.com/Azure/azure-sdk-for-java/pull/13229

I will merge the big PR tomorrow - and you can rebase this PR off of master and we can run integration tests to verify if everything looks good. 

Let me know if you have more questions on this.

cc: @jialindai

Reply:
ok, I'll rebase for master after you merge  #13229